packages/now-cli/src/util/dev/yarn-installer.ts

```diff
@@ -1,103 +0,0 @@
-import { createHash } from 'crypto';
-import {
-  mkdirp,
-  createWriteStream,
-  writeFile,
-  statSync,
-  chmodSync,
-  createReadStream,
-} from 'fs-extra';
-import pipe from 'promisepipe';
-import { join } from 'path';
-import fetch from 'node-fetch';
-import { Output } from '../output/create-output';
-import { builderDirPromise } from './builder-cache';
-
-const YARN_VERSION = '1.17.3';
```

Comment:
Is deleting this code fixing a bug?

I thought we want to mimic production which typically uses yarn?

Reply:
It's just removing unnecessary moving parts and will help `now dev` start up faster.

Note that since we no longer use `pkg`, it's safe to assume that `node` itself is installed (since it's required to run Now CLI), which comes bundled with `npm`.

Also note that the Runtimes (node, next, static-build) do not actually install user's dependencies when run in `now dev` (it's up to the user to do that), so invoking npm will only be required for installing Runtimes to the cache dir (and soon, only community runtimes that aren't bundled with now cli).

Reply:
Ah yes, very good points 👌